CVE-2017-9805/ubuntu2.yml

```diff
@@ -0,0 +1,17 @@
+- name: Linux target
+  hosts: all
+  become: true
+  tasks:
+    - name: Install requirements
+      ansible.builtin.apt:
+        update_cache: yes
+        name: python3
+        state: present
+    #- name: Git checkout
```

Comment:
Same here: please do not keep commented commands here. You may, however, mention the URL in human readable comment.